lizard.py

```diff
@@ -469,13 +469,18 @@ def __init__(self, ext):
             for caption, part, average in self._ext_member_info()]
         self.items.append({'caption': " location  ", 'value': 'location'})
 
+    @staticmethod
+    def get_stringtype():
+        try:
+            stringtype = basestring
+        except NameError:   # Not compatible with python 3
+            stringtype = str
+        return stringtype
+
     def _ext_member_info(self):
         for ext in self.extensions:
             if hasattr(ext, "FUNCTION_CAPTION"):
-                try:
-                    stringtype = basestring
-                except NameError:   # Not compatible with python 3
-                    stringtype = str
+                stringtype = OutputScheme.get_stringtype()
                 if isinstance(ext.FUNCTION_INFO_PART, stringtype):
```

Comment:
instead of get_stringtype, we can probably extract a is_string_instance(). What do you think?